src/main/java/com/project/jagoga/accommodation/application/AccommodationService.java

```diff
@@ -20,23 +23,42 @@ public Long saveAccommodation(Accommodation accommodation) {
         return accommodationRepository.save(accommodation).getAccommodationId();
     }
 
+    public AccommodationResponseDto updateAccommodation(Long accommodationId, AccommodationRequestDto accommodationRequestDto) {
+        Accommodation accommodation = accommodationRepository.findById(accommodationId)
+                .orElseThrow(() -> new NotExistAccommodationException());
```

Comment:
```suggestion
                .orElseThrow(NotExistAccommodationException::new); 
```
이부분 이렇게 메서드 참조로 변경할 수 있을것 같아요.